graphql/server.go

```diff
@@ -165,14 +167,34 @@ func (c *conn) handleSubscribe(id string, subscribe *subscribeMessage) error {
 		ctx = batch.WithBatching(ctx)
 
 		start := time.Now()
-		span, ctx := opentracing.StartSpanFromContext(ctx, "thunder.subscription")
+
 		c.logger.StartExecution(ctx, tags, initial)
-		current, err := e.Execute(ctx, c.schema.Query, nil, query)
+
+		output := &ComputationOutput{
+			Metadata: make(map[string]interface{}),
+		}
+
+		var middlewares []MiddlewareFunc
+		middlewares = append(middlewares, c.middlewares...)
```

Comment:
You could `middlewares := append(nil, c.middlewares...)` or maybe that confuses the type inferencer.